Puppetfile.pinned

```diff
@@ -57,8 +57,10 @@ mod 'simp-simp_selinux_policy',
   :tag => '1.1.0'
 
 mod 'simp-utils',
-  :git => 'https://github.com/simp/simp-utils',
-  :tag => '6.6.0'
+  :git => 'https://github.com/trevor-vaughan/simp-utils',
```

Comment:
This is no longer appropriate.  In addition this PR should wait for the rubygem-simp-cli PR to go through.